internal/controller/kyma/controller.go

```diff
@@ -59,19 +60,24 @@ const (
 	patchStatusError  event.Reason = "PatchStatus"
 )
 
+type ModuleStatusService interface {
```

Comment:
I suggest before we have a concrete plan to introduce Service layer, let's be cautious while naming some components as service, the scope of this `ModuleStatus` seems too limited to justify it as a service, I would rather name it as `ModuleStatusHandler` or something small scoped.

Reply:
Sure, renamed it to handler 👍